skimage/measure/_find_contours.py

```diff
@@ -184,17 +184,16 @@ def _assemble_contours(segments):
                 if tail_num > head_num:
                     # tail was created second. Append tail to head.
                     head.extend(tail)
-                    # remove all traces of tail:
-                    ends.pop(tail[-1])
+                    # Remove tail from the detected contours
                     contours.pop(tail_num, None)
                     # Update contour starts end ends
```

Comment:
"starts end ends" ? is in the comment here and in the else case. I assume this is a typo for "starts and ends"?

Reply:
Yes, it should be "Update starts and ends " or "Update starts and ends variables"